t/plugin/jwt-auth-vault.t

```diff
@@ -163,6 +148,13 @@ passed
 
 
 === TEST 4: sign a jwt and access/verify /secure-endpoint, fails as no secret entry into vault
+--- yaml_config
```

Comment:
We can share the config via extra_yaml_config
See https://github.com/apache/apisix/blob/71c256be81d95d56ea38f9874699801cbbad6a2e/t/plugin/ext-plugin/http-req-call.t#L47

Reply:
Resolved